pyproject.toml

```diff
@@ -28,6 +28,7 @@ dynamic = ["version"]
 dependencies = [
     "optimade[server] ~=0.22",
     "oteapi-core ~=0.4",
+    "oteapi-dlite ~=0.1",
```

Comment:
why is `oteapi-dlite ~=0.1` here and `oteapi-dlite ~=0.1.1` in the `dlite` section?

Reply:
It should instead be utilizing `>=` and `<`, so it should be `>=0.1.1,<1`. Either that or just `~=0.1`.

Reply:
Not necessarily. This is part of the OPTIMADE Python tools and is only necessary when testing. It is a standard output from this package, and should maybe be muted. Again - it is not necessary for this plugin.